test/SILGen/default_arguments.swift

```diff
@@ -272,3 +272,11 @@ func localFunctionWithDefaultArg() {
 }
 // CHECK-LABEL: sil private @_T017default_arguments27localFunctionWithDefaultArgyyF3barL_yySiSgFfA_
 // CHECK-SAME: $@convention(thin) () -> Optional<Int>
+
+func throwingDefault(closure: () throws -> Bool  = {  return true }) throws {
```

Comment:
How about a non-throwing test case, and an @escaping testcase (where there's no conversion, presumably)?